mergify_engine/rules/filter.py

```diff
@@ -287,3 +304,97 @@ def BinaryFilter(
             "and": all,
         },
     )
+
+
+DT_INFINITY = datetime.datetime.max.replace(tzinfo=datetime.timezone.utc)
```

Comment:
```suggestion
DT_MAX = datetime.datetime.max.replace(tzinfo=datetime.timezone.utc)
```
Infinity has 2 directions, it's unclear which one it is.